[2022-08-06 19:37:55,082] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-08-06 19:37:55,099] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-08-06 19:37:55,100] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:55,100] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:37:55,100] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:55,124] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-01-29 00:00:00+00:00
[2022-08-06 19:37:55,131] {standard_task_runner.py:52} INFO - Started process 1846 to run task
[2022-08-06 19:37:55,135] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-29T00:00:00+00:00', '--job-id', '558', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpip8pammd', '--error-file', '/tmp/tmpxmye01vt']
[2022-08-06 19:37:55,135] {standard_task_runner.py:80} INFO - Job 558: Subtask second_task
[2022-08-06 19:37:55,237] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:37:55,385] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-29T00:00:00+00:00
[2022-08-06 19:37:55,386] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:37:55,387] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:37:55,404] {subprocess.py:85} INFO - Output:
[2022-08-06 19:37:55,406] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:37:55,406] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:37:55,607] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220129T000000, start_date=20220806T193755, end_date=20220806T193755
[2022-08-06 19:37:55,670] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:37:55,717] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:51:46,758] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-08-06 21:51:46,771] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-08-06 21:51:46,771] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:46,771] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:51:46,771] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:46,788] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-01-29 00:00:00+00:00
[2022-08-06 21:51:46,795] {standard_task_runner.py:52} INFO - Started process 2022 to run task
[2022-08-06 21:51:46,798] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-29T00:00:00+00:00', '--job-id', '556', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpe2lvnjoa', '--error-file', '/tmp/tmpjsw1a62u']
[2022-08-06 21:51:46,799] {standard_task_runner.py:80} INFO - Job 556: Subtask second_task
[2022-08-06 21:51:46,885] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:51:46,976] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-29T00:00:00+00:00
[2022-08-06 21:51:46,977] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:51:46,977] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:51:46,993] {subprocess.py:85} INFO - Output:
[2022-08-06 21:51:46,995] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:51:46,996] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:51:47,040] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220129T000000, start_date=20220806T215146, end_date=20220806T215147
[2022-08-06 21:51:47,089] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:51:47,122] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:57:10,736] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-08-06 21:57:10,752] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-08-06 21:57:10,752] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:10,752] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:57:10,752] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:10,768] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-01-29 00:00:00+00:00
[2022-08-06 21:57:10,776] {standard_task_runner.py:52} INFO - Started process 1818 to run task
[2022-08-06 21:57:10,779] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-29T00:00:00+00:00', '--job-id', '554', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpko2_k15v', '--error-file', '/tmp/tmps50g2yx_']
[2022-08-06 21:57:10,779] {standard_task_runner.py:80} INFO - Job 554: Subtask second_task
[2022-08-06 21:57:10,851] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:57:10,944] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-29T00:00:00+00:00
[2022-08-06 21:57:10,945] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:57:10,945] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:57:10,959] {subprocess.py:85} INFO - Output:
[2022-08-06 21:57:10,960] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:57:10,960] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:57:10,993] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220129T000000, start_date=20220806T215710, end_date=20220806T215710
[2022-08-06 21:57:11,030] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:57:11,057] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:12:14,518] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-08-06 22:12:14,534] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-08-06 22:12:14,534] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:14,534] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:12:14,534] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:14,550] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-01-29 00:00:00+00:00
[2022-08-06 22:12:14,557] {standard_task_runner.py:52} INFO - Started process 1831 to run task
[2022-08-06 22:12:14,561] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-29T00:00:00+00:00', '--job-id', '556', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpnqdaj23w', '--error-file', '/tmp/tmpx7999kag']
[2022-08-06 22:12:14,562] {standard_task_runner.py:80} INFO - Job 556: Subtask second_task
[2022-08-06 22:12:14,651] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:12:14,738] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-29T00:00:00+00:00
[2022-08-06 22:12:14,739] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:12:14,740] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:12:14,752] {subprocess.py:85} INFO - Output:
[2022-08-06 22:12:14,754] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:12:14,754] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:12:14,790] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220129T000000, start_date=20220806T221214, end_date=20220806T221214
[2022-08-06 22:12:14,812] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:12:14,853] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:23:34,782] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-08-06 22:23:34,802] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [queued]>
[2022-08-06 22:23:34,802] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:34,802] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:23:34,802] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:34,821] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-01-29 00:00:00+00:00
[2022-08-06 22:23:34,829] {standard_task_runner.py:52} INFO - Started process 1888 to run task
[2022-08-06 22:23:34,832] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-29T00:00:00+00:00', '--job-id', '559', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpt506pmyc', '--error-file', '/tmp/tmpl06uxcch']
[2022-08-06 22:23:34,832] {standard_task_runner.py:80} INFO - Job 559: Subtask second_task
[2022-08-06 22:23:34,920] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-29T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:23:35,022] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-29T00:00:00+00:00
[2022-08-06 22:23:35,023] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:23:35,024] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:23:35,039] {subprocess.py:85} INFO - Output:
[2022-08-06 22:23:35,041] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:23:35,042] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:23:35,077] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220129T000000, start_date=20220806T222334, end_date=20220806T222335
[2022-08-06 22:23:35,124] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:23:35,183] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
